gatekeeper/http/http.go

```diff
@@ -28,11 +31,35 @@ type Gatekeeper struct {
 }
 
 // NewGatekeeper returns a new Gatekeeper.
-func NewGatekeeper(org, team string, cfg *config.Config, api *api.Client) *Gatekeeper {
+func NewGatekeeper(org, team, certpath, keypath string, cfg *config.Config, api *api.Client) (*Gatekeeper, error) {
 	server := &http.Server{
 		Addr: cfg.GatekeeperAddress,
 	}
 
+	if certpath == "" || keypath == "" {
```

Comment:
or better: have tlsKeypair error appropriately if both are not provided (or above someplace)..